tests/mirage/me/crate-owner-invitations/list-test.js

```diff
@@ -55,6 +55,7 @@ module('Mirage | GET /api/v1/me/crate_owner_invitations', function (hooks) {
           invited_by_username: 'janed',
           invitee_id: Number(user.id),
           inviter_id: Number(inviter.id),
+          expires_at: '2017-01-24T12:34:56Z',
```

Comment:
while this is generally useful, it is somewhat unrelated to the new mirage endpoint AFAICT, so it might be easier to put this in a dedicated PR.

we probably also would want to add this new field to the corresponding mirage "factory", so that all endpoints will always serialize the new field.

Reply:
I took this field from the struct here: https://github.com/rust-lang/crates.io/blob/8be845c314905f95fc972419c67eba57084dae2c/src/views.rs#L117

Noticed that is also returned by the controller here, https://github.com/rust-lang/crates.io/blob/3d712177dafbd972aa04177690a4aa8fcbfb53c2/src/controllers/crate_owner_invitation.rs#L42

I could introduce it when working on factories?

Reply:
sure, but what you linked is Rust code, which we don't run at all during the frontend tests, so on the mirage mock backend side there is currently no code that crates those fields

Reply:
Yeah, my goal was to be transparent with structure in the server. Perhaps I misunderstanding this field? But I wanted to mock the responses from server to match them with what Mirage will return. I could either remove this or update Mirage factories to include this, not sure to be honest now that I'm a bit confused.

Reply:
Removed the expires at field on [974476a](https://github.com/rust-lang/crates.io/pull/5397/commits/974476a2eba5cc8173cd694f71dd5cd39b4c25f1)